plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryTasksProvider.kt

```diff
@@ -0,0 +1,88 @@
+package io.sentry.android.gradle
+
+import com.android.build.gradle.api.ApplicationVariant
+import org.gradle.api.Project
+import org.gradle.api.Task
+
+internal object SentryTasksProvider {
+
+    /**
+     * Returns the transformer task for the given project and variant.
+     * It could be either ProGuard or R8
+     *
+     * @return the task or null otherwise
+     */
+    @JvmStatic
+    fun getTransformerTask(project: Project, variantName: String): Task? =
+        project.findTask(
+            // Android Studio 3.3 includes the R8 shrinker.
+            "transformClassesAndResourcesWithR8For${variantName.capitalized}",
+            "transformClassesAndResourcesWithProguardFor${variantName.capitalized}",
```

Comment:
I wonder if this is still necessary if we support AGP 4.0 and up only

Reply:
True, thought the CI matrix runs on AGP 3.4 and above so is probably worth to keep.